docs/information/supported_adapters.md

```diff
@@ -169,6 +169,15 @@ Zigbee2MQTT officially supports the following adapters:
     <td><a href="https://github.com/cyijun/OpenZ3Gateway">Description</a></td>
     <td><a href="mailto:cyjason2000@163.com">Contact</a></td>
   </tr>
+  <tr>
+   <tr>
+    <td><img src="../images/CC2652P-Z2M.jpg"></td>
```

Comment:
Please fix the picture, https://github.com/Koenkk/zigbee2mqtt.io/pull/873/files is just an empty file

Reply:
![cc2652P-Z2M](https://user-images.githubusercontent.com/24872854/130325459-01f8ca9e-c9ef-476e-9611-f0eb7db87290.jpg)
pic is there thanks